test/test_kl.jl

```diff
@@ -71,7 +71,7 @@ function test_subtract_kl()
     Celeste.DeterministicVI.subtract_kl_source!(sf, kl_result, vs, kl_helper)
     @test sf.v[] == DiffBase.value(kl_result)
     @test sf.d === DiffBase.gradient(kl_result)
-    test_sf = JLD.load(joinpath(dirname(@__FILE__), "kl_values.jld"), "sf")
+    test_sf = JLD.load(joinpath(datadir, "kl_values.jld"), "sf")
```

Comment:
@jrevels -- I moved your jld file to the data directory with the other jld test files. Hope that's OK.